src/v2/guide/computed.md

```diff
@@ -300,6 +295,6 @@ var watchExampleVM = new Vue({
 </script>
 {% endraw %}
 
-In this case, using the `watch` option allows us to perform an asynchronous operation (accessing an API), limit how often we perform that operation, and set intermediary states until we get a final answer. None of that would be possible with a computed property.
+Trong trường hợp này, sử dụng `watch` cho phép chúng ta thực hiện những tính toán không đồng bộ (ví dụ: truy cập tới một API) và giới hạn những tính toán đó, sau đó giá trị sẽ được gán ngay lập tức vào state khi có được đáp án cuối cùng. Những điều trên sẽ không thực hiện được nếu bạn sử dụng thuộc tính computed.
 
-In addition to the `watch` option, you can also use the imperative [vm.$watch API](../api/#vm-watch).
+Những thông tin thêm về `watch` có thể được tìm thấy tại [vm.$watch API](../api/#vm-watch).
```

Comment:
Dịch sai **hoàn toàn**. Người ta nói là "ngoài tùy chọn `watch` (trong cái object `options` ấy), thì bạn cũng có thể dùng `vm.$watch`".